.github/workflows/rotate-emoji.yml

```diff
@@ -0,0 +1,53 @@
+name: Rotate emojiData
```

Comment:
Minor nitpick, but "Update Emoji" (YAML `name`) and `update-emoji.yml` (file name) are probably better names since we aren't "rotating" emoji in/out.